addons/website/static/src/scss/website.scss

```diff
@@ -361,6 +361,15 @@ table.table_desc tr td {
     border-radius: 0;
 }
 
+// Allows custom border radius without contents overflowing.
+.card {
+    overflow: hidden;
+}
+
+.row > div > .card {
+    height: 100%;
+}
```

Comment:
This was my one of my ideas but don't forget to test them. As soon as you put a card inside an equal height column, this brokes the layout. I guess this cannot be a generic rule anyway. If someone drops a card in an empty column, he may want it full-height or not. Maybe it should be an option toggling the `h-100` class on cards? (and make the `s_three_columns` cards `h-100` by default)

Reply:
I think this would make much more sense yes. I did not know we had equal height columns were you could drop cards, but that seems to be the case of the footer and it also seems that you can configure the image-text snippets to be that way.

Reply:
(In general: all the standard row used in snippets are configured that way, but also the rule here would also apply on general layout in blog/event, etc).